flags.ts

```diff
@@ -7,80 +7,162 @@ import MatchRandom from "$live/functions/MatchRandom.ts";
 import MatchSite from "$live/functions/MatchSite.ts";
 import EffectSelectPage from "$live/functions/EffectSelectPage.ts";
 
+const DECO_COOKIE = "deco_flag_";
+
+export const cookies = {
+  parse: (rawCookies: string | null) => {
+    const flags: CookiedFlag[] | undefined = rawCookies
+      ?.split("; ")
+      .map((c) => c.split("="))
+      .filter((cookie) => cookie[0].startsWith(DECO_COOKIE))
+      .map((cookie) => JSON.parse(atob(cookie[1])));
```

Comment:
Suggestion: 

```ts
import { getCookies, setCookie } from "std/http/mod.ts";
```

https://deno.land/std@0.171.0/http/cookie.ts?s=getCookies

Reply:
I think this might not work because you're using `startsWith`, but it might be helpful for setting the cookie later

Reply:
yeah, I made it work now :) Thanks!